packages/cli/src/commands/setup/jobs/jobsHandler.js

```diff
@@ -0,0 +1,143 @@
+  runAt     DateTime?
+  lockedAt  DateTime?
+  lockedBy  String?
+  lastError String?
+  failedAt  DateTime?
+  createdAt DateTime  @default(now())
+  updatedAt DateTime  @updatedAt
+}
+`
+
+const getModelNames = async () => {
+  const schema = await getDMMF({ datamodelPath: getPaths().api.dbSchema })
+
+  return schema.datamodel.models.map((model) => model.name)
+}
+
+const addModel = () => {
+  const schema = fs.readFileSync(getPaths().api.dbSchema, 'utf-8')
+
+  const schemaWithUser = schema + MODEL_SCHEMA
+
+  fs.writeFileSync(getPaths().api.dbSchema, schemaWithUser)
+}
+
+const tasks = ({ force }) => {
+  let skipSchema = false
+
+  return new Listr(
+    [
+      {
+        title: 'Creating job model...',
+        task: async (_ctx, task) => {
+          if ((await getModelNames()).includes('BackgroundJob')) {
+            skipSchema = true
+            task.skip('Model already exists, skipping creation')
+          } else {
+            addModel()
+          }
+        },
+      },
+      {
+        title: 'Migrating database...',
+        task: async (_ctx, task) => {
+          if (skipSchema) {
+            task.skip('Model already exists, skipping migration')
+          } else {
+            execa.sync(
+              'yarn rw prisma migrate dev',
+              ['--name', 'create-background-jobs'],
+              {
+                shell: true,
+                cwd: getPaths().base,
+                stdio: 'inherit',
+              },
+            )
+          }
+        },
+      },
+      {
+        title: 'Creating config file in api/src/lib...',
+        task: async () => {
+          const isTs = isTypeScriptProject()
+          const outputExtension = isTs ? 'ts' : 'js'
+          const outputPath = path.join(
+            getPaths().api.lib,
+            `jobs.${outputExtension}`,
+          )
+          let template = fs
+            .readFileSync(
+              path.resolve(__dirname, 'templates', 'jobs.ts.template'),
+            )
+            .toString()
+
+          if (!isTs) {
+            template = await transformTSToJS(outputPath, template)
+          }
+
+          writeFile(outputPath, template, {
+            overwriteExisting: force,
+          })
+        },
+      },
+      {
+        title: 'Creating jobs dir at api/src/jobs...',
+        task: () => {
+          try {
+            fs.mkdirSync(getPaths().api.jobs)
+          } catch (e) {
+            // ignore directory already existing
+            if (!e.message.match('file already exists')) {
+              throw new Error(e)
+            }
+          }
+          writeFile(path.join(getPaths().api.jobs, '.keep'), '', {
+            overwriteExisting: force,
+          })
+        },
+      },
+      {
+        title: 'One more thing...',
+        task: (_ctx, task) => {
+          task.title = `One more thing...
+          ${c.green('Background jobs configured!\n')}
+          ${'Generate jobs with:'} ${c.warning('yarn rw g job <name>')}
```

Comment:
It's good to use a different color, to make it stand out more. But I don't think this is a warning. So I'd chose a different color.

Reply:
Yeah I don't love that we mapped colors to "danger levels"... I just wanted yellow/orange because it stands out, but doesn't scream. `chalk` itself just uses colors, but we decided to map those to names back in the day (but then we still kept "green" for some reason!):

```
export default {
  error: chalk.bold.red,
  warning: chalk.keyword('orange'),
  green: chalk.green,
  info: chalk.grey,
  bold: chalk.bold,
  underline: chalk.underline,
}
```

I could just add the list of actual colors to that export so it stays backwards compatible, and we can use them as intended.

Reply:
Added more color options in https://github.com/redwoodjs/redwood/pull/10939
I recommend `c.tip(...)` here